src/views/waypoint/home/components/cards/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+import InlineSvg from '@hashicorp/react-inline-svg'
+import classNames from 'classnames'
+import tagIconDict from './tag-icon-dict.json'
+import { CardProps } from './types'
+import s from './style.module.css'
+
+function Cards({ columns, cards }: CardProps): React.ReactElement {
+  return (
+    <div className={s.root}>
+      <div className={classNames(s.cards, s[`columns-${columns}`])}>
+        {cards.map((card, idx) => {
+          const { iconSvg, heading, text, tags } = card
+          return (
+            <div
+              // eslint-disable-next-line react/no-array-index-key
+              key={idx}
```

Comment:
Great callout.... tbh on the fence about this, as I'm worried `heading` might not be specific enough. Eg what if there are multiple sub-headings under broader heading sections, eg `## This one thing >> ### Install` & `## This other thing > ### Install` in the same `.mdx` doc... then I think we'd end up with identical `key` values, which I think is more dangerous than index-as-key? (esp since we're not expecting to re-render or re-order this content, I think?)

Let me know if that makes any sense, I feel like I've ran into this issue with not-guaranteed-unique keys before, but not sure 😬 🙇